.github/workflows/docker.yml

```
@@ -0,0 +1,51 @@
+name: Build and publish Docker image
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Build Docker image
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```
This action can be updated.